tests/migration/sle12_online_migration/yast2_migration.pm

```diff
@@ -203,9 +203,8 @@ sub run {
     }
     assert_screen 'yast2-migration-target';
     send_key "alt-p";                                       # focus on the item of possible migration targets
-    send_key_until_needlematch 'migration-target-' . get_var("VERSION"), 'down', 5;
-    wait_still_screen 5;
-    send_key "alt-n";
+    send_key_until_needlematch 'migration-target-' . get_var("VERSION"),                      'down',  5;
+    send_key_until_needlematch ['yast2-migration-installupdate', 'yast2-migration-proposal'], 'alt-n', 4, 5;
     assert_screen ['yast2-migration-installupdate', 'yast2-migration-proposal'], 500;
```

Comment:
This `send_key_until_needlematches` probably makes the `assert_screen` call redundant.